apps/dashboard/src/components/truncated-text.tsx

```diff
@@ -26,11 +26,19 @@ export default function TruncatedText(props: TruncatedTextProps) {
     <Tooltip>
       <TooltipTrigger asChild>
         {asChild ? (
-          <Slot ref={textRef} className={cn('block truncate', className)} {...rest}>
+          <Slot
+            ref={textRef}
+            className={cn('truncate', isTruncated && 'block', !isTruncated && 'inline-flex', className)}
```

Comment:
Ensures that truncated texts use block, whilst non-truncated use inline-flex to allow short names to display inline

Reply:
Lovey touch.